src/components/Contexts/FormContext.tsx

```diff
@@ -124,15 +125,16 @@ export const FormProvider: FC<ProviderProps> = (props) => {
     // Update the state when the lazy query response changes
     if (data) {
       const applicationData = data?.getApplication;
+      const newData: Application = {
+        ...merge(initialValues, applicationData),
```

Comment:
Can we swap the argument order for this Lodash merge? It's updating the InitialValues variable which will combine values from different forms as you go through them.

![image](https://github.com/CBIIT/crdc-datahub-ui/assets/38357871/ed4f4983-c609-48a8-9656-4bcdc468ca30)

Reply:
Lodash is weird. I updated to make a deep copy of the initialValues object like you suggested. Thanks Alec!